service/src/integrationTest/java/org/databiosphere/workspacedataservice/service/BackupServiceIntegrationTest.java

```diff
@@ -0,0 +1,26 @@
+package org.databiosphere.workspacedataservice.service;
+
+import org.databiosphere.workspacedataservice.storage.LocalFileStorage;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+
+import java.io.File;
+import java.util.Scanner;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@SpringBootTest(properties = "spring.cache.type=NONE")
+public class BackupServiceIntegrationTest {
+    @Autowired
+    private BackupService mockBackupService;
+
+    private LocalFileStorage storage = new LocalFileStorage();
+
+    @Test
+    void testBackupAzureWDS() throws Exception {
+        var response = mockBackupService.backupAzureWDS(storage, "v0.2");
+        assertTrue(response.backupStatus());
```

Comment:
nit: include the backup error message in the cases this test fails, that will make test-debugging a lot easier:
```suggestion
        assertTrue(response.backupStatus(), response.message());
```